istio/create-authorizationpolicy/02-clusterrole.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: kyverno:background-controller:additional
```

Comment:
Here and in subsequent tests where we need to add custom permissions, don't use the "additional" suffix. Make it meaningful for each test. For example, here you could the suffix "istio-authzpolicy" or something. The more specific and less generic we make any of the resources we create, the easier it is to track down issues in the kuttl output log or the Pod logs. If multiple tests are using the same generic names, if we have to go searching through logs it can be much more time consuming and difficult.

Reply:
Makes sense. I'll change it.